pkg/skaffold/build/cluster/pod_test.go

```diff
@@ -233,6 +233,7 @@ func TestKanikoPodSpec(t *testing.T) {
 
 	expectedPod := &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
+			Annotations:  map[string]string{"iam.amazonaws.com/role": "skaffold"},
```

Comment:
Let's just use `test: test` — it's best not to use live annotation strings just in case this is copied into a live integration test.

Reply:
Changed It . 👍